app/assets/stylesheets/reviews.scss

```diff
@@ -1,3 +1,5 @@
+@import "variables";
```

Comment:
you shouldn't need this import - application.scss should have `variables.scss` imported